src/pal/inc/pal.h

```diff
@@ -2829,7 +2832,7 @@ VOID
 PALAPI
 RtlMoveMemory(
           IN PVOID Destination,
-          IN CONST VOID *Source,
+          IN const VOID *Source,
```

Comment:
This can say upper case `CONST`

Reply:
Or deleted completely (from here and from the implementation). It is not used anywhere.

(can be done as a separate PR)

Reply:
I agree, it would be better as a separate PR. There are some usages under tests, that would probably need to be cleaned out as well:

```powershell
~\source\repos\coreclr [gcc-visibility ≡]> git grep RtlMoveMemory
src/pal/src/memory/heap.cpp:  RtlMoveMemory
src/pal/src/memory/heap.cpp:RtlMoveMemory(
src/pal/src/memory/heap.cpp:    PERF_ENTRY(RtlMoveMemory);
src/pal/src/memory/heap.cpp:    ENTRY("RtlMoveMemory(Destination:%p, Source:%p, Length:%d)\n",
src/pal/src/memory/heap.cpp:    LOGEXIT("RtlMoveMemory returning\n");
src/pal/src/memory/heap.cpp:    PERF_EXIT(RtlMoveMemory);
src/pal/tests/palsuite/filemapping_memmgt/CMakeLists.txt:add_subdirectory(RtlMoveMemory)
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test1/test1.cpp:    RtlMoveMemory(NewAddress, OldAddress, MEMORY_AMOUNT);
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test1/testinfo.dat:Function = RtlMoveMemory
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test1/testinfo.dat:Name = Positive test for RtlMoveMemory API
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test3/test3.cpp:    RtlMoveMemory(NewAddress, OldAddress, OLD_MEMORY_AMOUNT);
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test3/testinfo.dat:Function = RtlMoveMemory
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test3/testinfo.dat:Name = Positive test for RtlMoveMemory API, move a small block to large
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test4/test4.cpp:    RtlMoveMemory(NewAddress, OldAddress, 50);
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test4/testinfo.dat:Function = RtlMoveMemory
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test4/testinfo.dat:Name = Positive test for RtlMoveMemory API, test that overlapping works
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test5/test5.cpp:    RtlMoveMemory(NewAddress, SectionToMove, MEMORY_AMOUNT-50);
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test5/testinfo.dat:Function = RtlMoveMemory
src/pal/tests/palsuite/filemapping_memmgt/RtlMoveMemory/test5/testinfo.dat:Name = Positive test for RtlMoveMemory API, test that overlapping works
src/pal/tests/palsuite/paltestlist.txt:filemapping_memmgt/RtlMoveMemory/test1/paltest_rtlmovememory_test1
src/pal/tests/palsuite/paltestlist.txt:filemapping_memmgt/RtlMoveMemory/test3/paltest_rtlmovememory_test3
src/pal/tests/palsuite/paltestlist.txt:filemapping_memmgt/RtlMoveMemory/test4/paltest_rtlmovememory_test4
src/pal/tests/palsuite/paltestlist.txt:filemapping_memmgt/RtlMoveMemory/test5/paltest_rtlmovememory_test5
```

Reply:
These are tests for `RtlMoveMemory`. They should be delete with it.